src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+
+    protected String dlDate;
```

Comment:
could use less abbreviation i.e. deadlineDate

Reply:
Ok thanks for the suggestion!